### DISCUSSION TITLE
Difficulty accessing assignments Cengage/Github

### DISCUSSION THREAD

**Discussion by gibroons** [community/community#103056] [New to GitHub]
I have been trying to access my cengage coding exercises for weeks with no success. I log into Cengage/Mindtap and I'm able to do the readings but any coding exercises are inaccessible. I click on the exercise and it tells me "This is your first attempt" for the exercise. I launch the exercise and a pop up window appears. It says "Authorize Cengage IDE Coding Labs 05". I click on the authorize button and it redirects me to the authorized application but it just buffers until it says "504 Gateway timeout". Please advise on how this can be fixed. I haven't been able to do any of my coding assignments as a result of this issue. I don't have any adblockers or pop up blockers either. If there's anyone that can help guide me in this situation I would be very grateful.
<img width="1432" alt="Screen Shot 2024-01-24 at 12 01 28 PM" src="https://github.com/community/community/assets/90077348/73e32c91-ca51-466d-abc5-919399781afe">
<img width="1436" alt="Screen Shot 2024-01-24 at 12 04 25 PM" src="https://github.com/community/community/assets/90077348/ecf443ac-8d35-4207-b421-d6a6dc6996b1">
<img width="1437" alt="Screen Shot 2024-02-01 at 2 41 17 PM" src="https://github.com/community/community/assets/90077348/0cf0a07f-1b1c-4a7a-a625-7264a40eb121">
<img width="1431" alt="Screen Shot 2024-02-01 at 2 41 27 PM" src="https://github.com/community/community/assets/90077348/c9a2fa6d-215a-4241-aa00-434b062e6f2e">
<img width="1435" alt="Screen Shot 2024-02-01 at 2 43 09 PM" src="https://github.com/community/community/assets/90077348/76beb424-6ec7-406d-8979-38189193a2cc">
<img width="1433" alt="Screen Shot 2024-02-01 at 4 05 00 PM" src="https://github.com/community/community/assets/90077348/10e28b56-3220-472c-8476-28dbcce5bfcf">

**Comment by nmciavarella** [comment]
I teach at a technical school using Cengage curriculum. We are on a normal four quarters, 9 week each schedule. Without notice over the students 1 week holiday break Cengage completely flipped the digital lab component to a broken GitHub based platform. It requires students to create github accounts, and largely appears half developed, and is mostly broken once you manage to access labs.

I would definitely reach out to your instructor, they may not even know the lab has changed and is effectively broken. We received 0 communication from them, it's been a month and none of the labs work still.

I just started using my existing lessons and labs and decided to drop this broken curriculum.

Cengage is definitely kicked out of our classroom.

**Reply by ghost** [reply]
Even before the Github frustration, Cengage has been a disappointment.

**Comment by cris-renee** [comment]
Hi gibroons 
I had a time getting connected as well. I couldn’t tell u what I did to eventually connect but it did. Have u checked (and changed if needed)  the firewall ? I just kept fooling around with mine & something worked lol. I am currently having trouble submitting assignments. They need to get this fixed ASAP or have a list of detailed instructions on how to do everything. Has others been having problems or do you know?

**Comment by ghost** [comment]
I have been having the same issue as well I have been messing with it for days please help
